x-pack/metricbeat/module/prometheus/collector/histogram.go

```diff
@@ -0,0 +1,52 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package collector
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/elastic/beats/v7/libbeat/common"
+
+	dto "github.com/prometheus/client_model/go"
+)
+
+// promHistogramToES takes a Prometheus histogram and converts it to an ES histogram
+func promHistogramToES(cc CounterCache, name string, labels common.MapStr, histogram *dto.Histogram) common.MapStr {
```

Comment:
Maybe it worths adding an example  in this docsstring of how a histogram in ES format looks like?